### PR TITLE
Alert Migration, Schema and Model Changes

### PR DESCRIPTION
**Pull request by vikramjadon921995**
# Why
The following changes were required to create the schema, migration and model for alerts.

## Changes
Added Alerts app directory structure, migration and schema
Added schema test cases for alerts
Added alert model and its test cases

### CODE REVIEW

File: apps/acqdat_core/priv/repo/migrations/20200714021956_create_alerts.exs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+defmodule AcqdatCore.Repo.Migrations.CreateAlerts do
+  use Ecto.Migration
+
+  def change do
+    create table(:alerts) do
+      add :name, :string
```
Comment (BandanaPandey) [inline apps/acqdat_core/priv/repo/migrations/20200714021956_create_alerts.exs:6]: @vikramjadon921995  are we not planning to make alert name unique ?
Correct me If I am wrong?

Comment (vikramjadon921995) [reply apps/acqdat_core/priv/repo/migrations/20200714021956_create_alerts.exs:6]: @BandanaPandey . A lot of alerts can be created by the same name. For eg: An alert for user creation can be created multiple times.

File: apps/acqdat_core/lib/acqdat_core/alerts/schema/alert.ex [hunk 2]
```diff
@@ -0,0 +1,50 @@
+defmodule AcqdatCore.Alerts.Schema.Alert do
+  use AcqdatCore.Schema
+
+  schema "alerts" do
+    field(:name, :string, null: false)
+    field(:description, :string)
+    field(:policy_name, PolicyDefinitionEnum)
+    field(:policy_module_name, PolicyDefinitionModuleEnum)
+    field(:app, AppEnum)
+    field(:entity_name, :string)
```
Comment (arjun289) [inline apps/acqdat_core/lib/acqdat_core/alerts/schema/alert.ex:10]: I would suggest putting a `validate_inclusion` clause here. You can define a list where you can put valid strings acceptable here.

**Comment by arjun289** [comment]
Already included in a separate PR

**Pull request closed without merging**